clevertap-core/src/main/java/com/clevertap/android/sdk/pushnotification/INotificationRenderer.java

```diff
@@ -148,8 +149,13 @@ default NotificationCompat.Builder setActionButtons(
                         actionIntent = PendingIntent.getService(context, requestCode,
                                 actionLaunchIntent, flagsActionLaunchPendingIntent);
                     } else {
+                        Bundle optionsBundle = null;
+                        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
+                            optionsBundle = ActivityOptions.makeBasic().setPendingIntentBackgroundActivityStartMode(
+                                    ActivityOptions.MODE_BACKGROUND_ACTIVITY_START_ALLOWED).toBundle();
+                        }
                         actionIntent = PendingIntent.getActivity(context, requestCode,
-                                actionLaunchIntent, flagsActionLaunchPendingIntent);
+                                actionLaunchIntent, flagsActionLaunchPendingIntent, optionsBundle);
```

Comment:
is this method also available in api33?

Reply:
Yess, getActivity is available. If the BUILD_VERSION is less than 33 then optionsBundle is null which is treated as no options by this method

<img width="416" alt="Screenshot 2024-01-29 at 11 36 56 AM" src="https://github.com/CleverTap/clevertap-android-sdk/assets/127097095/01dcaf1f-7c32-4e5f-ae1b-4b8fe2ad4957">